### DISCUSSION TITLE
UI Showcase

### DISCUSSION THREAD

**Discussion by Supernova3339** [cjmellor/level-up#73] [Show and tell]
![image](https://github.com/cjmellor/level-up/assets/63515814/4779f252-ca01-44a4-8f7a-8e680d21589c) development
![image](https://github.com/cjmellor/level-up/assets/63515814/33af187e-72fa-43ed-832d-1d19172013c7) production
![image](https://github.com/cjmellor/level-up/assets/63515814/0d31596f-3072-452b-9a29-85f72949ff48) light variant

the entire view and code is done with tailwindcss & livewire 3, having light/dark mode variants

When hovered ( on an achievement )
![image](https://github.com/cjmellor/level-up/assets/63515814/3589ce87-b0ed-4a02-b383-b68c6f952c06)

Each achievement has information that opens up via model ( Flowbite )
![image](https://github.com/cjmellor/level-up/assets/63515814/70d4ea56-9ada-4e08-a186-298f2ca768ee)

Will post some more if your interested 👀

**Comment by cjmellor** [comment]
Nice one. Good to see this package is in a state where folk can build stuff like this 😅

**Reply by Supernova3339** [reply]
Yeah haha

**Reply by abanghendri** [reply]
> Yeah haha

can I modify level message like "Beginer" instead of 1

**Comment by Supernova3339** [comment]
The message is just getting pulled from level name in the DB, just name it
whatever. What I did is a seeder that generated 100 levels
